src/DayColumn.js

```diff
@@ -97,7 +97,7 @@ class DayColumn extends React.Component {
     const current = getNow()
 
     if (current >= min && current <= max) {
-      const { top } = this.slotMetrics.getRange(current, current)
+      const top = this.slotMetrics.getCurrentTimePosition(current, current)
```

Comment:
Think we can trim this even more, since the start and end are always the same here, have it just take `current` and alter the range func accordingly?